### DISCUSSION TITLE
Hard to use input for workflow on mobile

### DISCUSSION THREAD

**Discussion by bogosj** [community/community#10104] [Actions]
When manually triggering a workflow on mobile, the input box renders off the left side of the screen. See an example from the chrome dev tools:

![image](https://user-images.githubusercontent.com/10186620/149430034-8d33fe48-7cba-4167-9a79-e362b2935d3f.png)

**Comment by ethomson** [comment]
Ouch!  I've asked the team to take a look, thanks for letting us know.
